static/js/dashboards-home.js

```diff
@@ -253,7 +253,15 @@ class btnRenderer {
                                 details: {
                                     name: duplicatedDBName,
                                     description: duplicatedDescription,
-                                    panels: duplicatedPanels,
+                                    panels: duplicatedPanels.map((panel) => ({
```

Comment:
When you duplicate a previously created dashboard, it gives an error. The same issue occurs if you duplicate a previously created panel.